conf/minion

```diff
@@ -270,6 +270,14 @@
 # often lower this value
 #loop_interval: 60
 
+# Some installations choose to start all job returns in a cache or a returner
+# and forgo sending the results back to a master. In this workflow, jobs
+# are most often executed with --async from the Salt CLI and then results
+# are evaluated by examining job caches on the minions or any configured returners.
+# WARNING: Setting this to False will **disable** returns back to the master.
+#pub_ret: True
```

Comment:
Mind adding this description to the `/doc/ref/configuration/minion.rst` page too?